### PR TITLE
feat: add "edit an application" post

### PR DESCRIPTION
**Pull request by thlmenezes**
## Description of change

1. Creates draft post "Edit an Application"

## Issues resolved by this PR

- [x] [DVN-12672]
- [x] [DVN-12673]

## More info

Post Preview
image

[DVN-12672]: https://devopness.atlassian.net/browse/DVN-12672?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DVN-12673]: https://devopness.atlassian.net/browse/DVN-12673?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/applications/edit-application.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+draft: true
+title: Edit an Application
+intro: Editing applications is crucial for ensuring accurate and up-to-date information is included in the resource deployment process, as needed. Learn how to edit an application's repository.
```
Comment (jfoliveira) [inline docs/applications/edit-application.md:4]:
```suggestion
intro: Learn how to edit an application to keep its configuration up to date, specially the settings that define how the application is deployed, such as source provider, repository, dependencies to be installed, directories and entrypoint.
```

**Review by jfoliveira — CHANGES_REQUESTED** [review]

File: docs/applications/edit-application.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+draft: true
```
Comment (jfoliveira) [inline docs/applications/edit-application.md:2]: why draft?

**Review by jfoliveira — CHANGES_REQUESTED** [review]

File: docs/applications/edit-application.md (hunk 2 above)
Comment (thlmenezes) [reply docs/applications/edit-application.md:2]: This post's goal (see Jira issue) is to teach how to update an application repository, an option currently not available in the web app.

Comment (jfoliveira) [reply docs/applications/edit-application.md:2]: If we just remove the steps below, and remove it from draft, the post will already be useful and might encourage other users to contribute to improve it, or am I missing something?

```md
1. Select a connected `Source Provider` account
    - If you want to use a different one, click `Connect a new source provider account` and follow the prompts
1. Use the drop-down menu to choose the repository
```

Comment (jfoliveira) [reply docs/applications/edit-application.md:2]: But anyway, if you wanna keep it in draft, that's fine. I'm approving anyway.

**Review by jfoliveira — APPROVED** [review]

File: docs/applications/edit-application.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    featured:
+required_permissions:
+    - application:update
+---
+
+1. On Devopness, navigate to a project then select an environment
+1. Find the `Applications` card
+1. Click `View` in the `Applications` card to see a list of existing `Applications`
+1. Click `DETAILS` on the application you want to edit
+1. On the upper roght corner of the application details view click `...`
```
Comment (thlmenezes) [inline docs/applications/edit-application.md:20]:
```suggestion
1. On the upper right corner of the application details view click `...`
```

**Review by jfoliveira — APPROVED** [review]

**Pull request merged**